lib/std/os/windows.zig

```diff
@@ -593,6 +594,8 @@ pub const CreateSymbolicLinkError = error{
     Unexpected,
 };
 
+/// Needs `SeCreateSymbolicLinkPrivilege` privilege, otherwise fails with `error.AccessDenied`.
```

Comment:
Could you add a comment stating that needs either `SeCreateSymbolicLinkPrivilege` *or* the OS in Developer Mode?

Reply:
Sure.